src/bootstrap.js

```diff
@@ -26,9 +27,9 @@ export default function() {
   }
 
   if (window[globalObjectName] == null) {
-    window.dataLayer = window.dataLayer || [];
+    window[globalDataLayerName] = [];
```

Comment:
this one should be `window[globalDataLayerName] = window[globalDataLayerName] || []`